package.json

```diff
@@ -8,6 +8,9 @@
   ],
   "private": true,
   "main": "src/web/index.js",
+  "config": {
+    "GA_TRACKING_ID": "UA-114768519-1"
```

Comment:
This is a temporary ID for a dashboard I made on my account. We should change it for production and possibly make it switch between IDs for prod / dev builds.

Reply:
Should we file a new issue to change this before release? Sadly it's in package.json so we can't add a `// TODO` comment.

Reply:
I'm waiting on a final name, and then will get a real GA ID and file an issue to implement it

Reply:
> Sadly it's in package.json so we can't add a `// TODO` comment.

And @linuxwolf has worked very hard to keep it that way 😉

Reply:
https://github.com/mozilla/Themer/issues/142 for the final ID